lib/laa_crime_schemas/structs/partner.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module LaaCrimeSchemas
+  module Structs
+    class Partner < Person
+      attribute? :date_of_birth, Types::JSON::Date.optional
+      attribute? :benefit_type, Types::BenefitType.optional
+      attribute? :last_jsa_appointment_date, Types::JSON::Date.optional
```

Comment:
I have a vague memory of Chris saying this wasn't collected
Not a blocker, but worth checking / tidying

Reply:
Chris has confirmed we do need to collect it